src/constants/fuseTokenMapping.ts

```diff
@@ -1,52 +1,57 @@
-export const getFuseTokenLogoURL = (address: string) => {
+export const getFuseTokenLogoURL = (address: string): string => {
     let uri
+    // TODO - addresses for prod to be updated
+    // dev=fuse staging=staging production=production
+    const G$Addresses = [
+        '0xeE724540706296ebad65aeA2515Efe0949F97Ae6', // dev
+        '0x100b8fd10ff8DC43fda45E636B4BB1eE6088270a' // staging
+    ].map(address => address.toLowerCase())
+    const GoodAddresses = [
+        '0x0Fce4a964F2b69a6cD82c3FB40C101863091A5a7', // dev
+        '0xf4467Ad0c9D7D0446B379fE6217111a4690066D1' // staging
+    ].map(address => address.toLowerCase())
 
     if (address?.toLowerCase() === '0x495d133B938596C9984d462F007B676bDc57eCEC'.toLowerCase()) {
```

Comment:
this is fuse production
for ethereum production simply search etherscan for GoodDollar

Reply:
@sirpy i didn't follow here can you elaborate a little?

Reply:
0x495d133B938596C9984d462F007B676bDc57eCEC this is fuse production G$ address
you should have an array for G$X GOOD and G$ with all envs addresses and not in a separate if like in line 14.

you wrote in TODO that you are missing production data
you can get all production addresses from the latest GoodDollar/GoodProtocol/release/deployment.json

Reply:
Thank you i will update it

Reply:
@mukulshakya well?

Reply:
@sirpy Updated